### ISSUE TITLE
feature parity with other video editors?

### ISSUE THREAD

**Issue by rudolfolah** [owickstrom/komposition#4]
Just curious to know if you're aiming for feature parity with OpenShot or kdenlive or any other video editor? It's been disappointing to use C/C++-based video editors that crash randomly or behave in weird ways!

**Comment by owickstrom** [comment]
@rudolfolah I haven't used OpenShot, but I have some experience with kdenlive.

I think Komposition should focus on doing only the basic screencast editing part, and do that really well, instead of going towards a more general video editor. Things like transitions and effects, multi-track video and audio, audio level automation, etc, all seems out of scope at this point. That said, I've thought about how it could integrate with other editors. For example, I'd personally like to edit the overall screencast in Komposition first, then export it to, say, a Kdenlive project where I can make the final touches.

Regarding the crashes, I agree, it's super frustrating. During the short period of using Komposition when developing it, I've found it very stable so far. The majority of problems have been around invoking external programs (ffmpeg, sox) correctly. I would love to have a more general video editor _also_ written in Haskell. 😄

**Comment by rudolfolah** [comment]
>Things like transitions and effects, multi-track video and audio, audio level automation, etc, all seems out of scope at this point.

True, for my own screencasting the thing I've found helpful with multi-track video is that it's possible for me to edit out the date/time from the taskbar on the screen. I think no multi-track video is good if it's possible to layer rectangles to hide things like that.

>I would love to have a more general video editor also written in Haskell. 😄

It's interesting to see whether the amount of time spent on bugs will be lessened through the use of Haskell.
